helpers.js

```diff
@@ -110,6 +109,17 @@ function fixClass(cls) {
   return cls;
 }
 
+function removePseudoClass(cls) {
+  const result = cls.replace(
+    /(:(active|after|before|checked|disabled|focus|focus-within|hover|visited|nth-child\((even|odd)\)|(first|last)-child))+$/,
+    ""
+  );
+
+  // Remove multiple by doing recursion when the result differs
+  if (result !== cls) return removePseudoClass(result);
```

Comment:
When I remove this line none of the tests fail - the regex is already grabbing chained selectors with `+` - do you have any other cases where the regex isn't enough?

Reply:
Yeah, you're totally right. Removed these changes 👍